src/pages/junghyowon/Main/JungMain.scss

```
@@ -0,0 +1,344 @@
+  text-align: center;
+
+  img {
+    width: 25px;
+    height: 25px;
+    object-fit: cover;
+    margin: 20px auto;
+  }
+
+  .logo {
+    margin: 40px auto;
+  }
+
+  .myprofile {
+    border-radius: 50%;
+  }
+
+  .moreview-img {
+    display: flex;
+    align-items: flex-end;
+    justify-content: end;
+    height: 100%;
+  }
+}
+
+.mainfeed-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+
+  article {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 10px;
+
+    .mainfeed-top {
+      width: 450px;
+      height: 115px;
+      margin: 20px auto auto auto;
+      padding: 15px;
+      border: 1px solid lightgray;
+      border-radius: 8px;
+      background-color: #fff;
+
+      .feed-profile {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: flex-start;
+
+        img {
+          width: 55px;
+          height: 55px;
+          border-radius: 50%;
+        }
+
+        span {
+          margin-top: 8px;
+          font-size: 12px;
+        }
+
+        .feed-profile1 {
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          justify-content: center;
+          margin: 5px 13px 5px 0px;
+        }
+      }
+    }
+
+    .feed-image-top {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      width: 450px;
+      height: 65px;
+      margin: 15px auto auto auto;
+      padding: 10px;
+      border: 1px solid lightgray;
+      border-radius: 8px 8px 0px 0px;
+      background-color: #fff;
+
+      .feed-image-top-more {
+        display: flex;
+        justify-content: end;
+        width: 500px;
+
+        img {
+          width: 23px;
+          min-width: 23px;
+          height: 23px;
+        }
+      }
+
+      img {
+        width: 30px;
+        min-width: 30px;
+        height: 30px;
+        border-radius: 50%;
+        object-fit: cover;
+      }
+
+      span {
+        margin-left: 10px;
+        font-size: 14px;
+      }
+    }
+
+    .feed-image {
+      width: 450px;
+      height: 580px;
+      margin: 0 auto;
+
+      img {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        overflow: hidden;
+      }
+    }
+
+    .mainfeed-comment-top {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      width: 450px;
+      border-right: 1px solid lightgray;
+      border-left: 1px solid lightgray;
+      background-color: #fff;
+
+      img {
+        width: 25px;
+        height: 25px;
+        margin: 10px;
+      }
+
+      .bookmark {
+        display: flex;
+        justify-content: end;
+        width: 500px;
+
+        img {
+          width: 23px;
+          height: 23px;
+        }
+      }
+    }
+
+    .feed-comment-main {
+      width: 450px;
+      border-right: 1px solid lightgray;
+      border-bottom: 1px solid lightgray;
+      border-left: 1px solid lightgray;
```

```suggestion
      border: 1px solid lightgray;
      border-top : none;
```
이렇게 두줄로 줄일 수 있을거 같아요